.github/workflows/platformDeploy.yml

```diff
@@ -374,9 +374,11 @@ jobs:
   postSlackMessageOnSuccess:
     name: Post a Slack message when all platforms deploy successfully
     runs-on: ubuntu-latest
-    if: ${{ success() }}
     needs: [android, desktop, iOS, web]
     steps:
+      - name: Set version
+        run: echo "VERSION=$(npm run print-version --silent)" >> "$GITHUB_ENV"
```

Comment:
You'll need to run `actions/checkout` before this will work

Reply:
Updated!